lib/Header/Header.js

```diff
@@ -305,7 +305,8 @@ module.exports = kind(
 		{name: 'titleWrapper', kind: Control, classes: 'moon-header-title-wrapper', components: [
 			{name: 'title', kind: MarqueeText, classes: 'moon-header-text moon-header-title', canGenerate: false},
 			{name: 'inputDecorator', kind: InputDecorator, classes: 'moon-input-header-input-decorator',canGenerate: false, components: [
-				{name: 'titleInput', kind: Input, classes: 'moon-header-text moon-header-title'}
+                {name: 'titleInput', kind: Input, classes: 'moon-header-text moon-header-title',  onblur: 'inputBlur'},
+                {name: 'titleInputMarquee', kind: MarqueeText, classes: 'moon-header-text moon-header-title moon-input header-marquee hidden', marqueeOnRender: true, ontap: "marqueToggle"}
```

Comment:
There's no need for the `hidden` class as we have a `showing` property that can be set for Controls.